test/helpers/docker.go

```diff
@@ -32,9 +32,10 @@ func (s *SSHMeta) ContainerExec(name string, cmd string, optionalArgs ...string)
 
 // ContainerCreate is a wrapper for `docker run`. It runs an instance of the
 // specified Docker image with the provided network, name, and options.
-func (s *SSHMeta) ContainerCreate(name, image, net, options string) *CmdRes {
+func (s *SSHMeta) ContainerCreate(name, image, net, options string, cmdOnStart string) *CmdRes {
```

Comment:
Could you move this cmdOnStart as variadic argument? It's only used one time and you had changed how the function works, I'll do something like this: 
```
func (s *SSHMeta) ContainerCreate(name, image, net, options string, cmdParams ...string) *CmdRes {
	cmdOnStart := ""
	if len(cmdParams) > 0 {
		cmdOnStart = strings.Join(cmdParams, " ")
	}

	cmd := fmt.Sprintf(
		"docker run -d --name %s --net %s %s %s %s", name, net, options, image, cmdOnStart)
	log.Debugf("spinning up container with command '%v'", cmd)
	return s.ExecWithSudo(cmd)
}
```

This allow to keep other functions as it was, and no new argument at all. In the case that you need to run a specific command can be run with the variadic argument.

Reply:
sure

Reply:
done